setup.py

```diff
@@ -73,6 +73,9 @@ class OpenLDAP2:
   author = 'python-ldap project',
   author_email = 'python-ldap@python.org',
   url = 'https://www.python-ldap.org/',
+  project_urls = {
+    'Source': 'https://github.com/python-ldap/python-ldap',
+  },
```

Comment:
Good catch!
Could you please, also, include the link to our docs?

```
+  project_urls = {
+    'Documentation': 'https://www.python-ldap.org/en/latest/',
+    'Source': 'https://github.com/python-ldap/python-ldap/',
+  },
```

Reply:
url already pointing to docs